app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/dlc/DLCStatus.scala

```diff
@@ -198,12 +218,16 @@ object DLCStatus {
       val outcomeValues = cet.outputs.map(_.value).sorted
       val totalCollateral = offer.totalCollateral + accept.totalCollateral
 
-      val possibleMessages = offer.contractInfo.filter {
-        case (_, amt) =>
-          Vector(amt, totalCollateral - amt)
-            .filter(_ >= Policy.dustThreshold)
-            .sorted == outcomeValues
-      }.keys
+      val possibleMessages = offer.contractInfo match {
+        case DLCMessage.SingleNonceContractInfo(outcomeValueMap) =>
+          outcomeValueMap.filter {
+            case (_, amt) =>
+              Vector(amt, totalCollateral - amt)
+                .filter(_ >= Policy.dustThreshold)
+                .sorted == outcomeValues
+          }.keys
+        case DLCMessage.MultiNonceContractInfo(_, _, _, _) => ???
```

Comment:
ping

Reply:
Might have to just throw an error here or change the return type of this because oracle sig won't be able to be calculated